simple_use_case/cwl/README.md

```diff
@@ -20,6 +20,13 @@ cwltool wf_run_use_case.cwl
 
 into your terminal.
 
+Or you can use the conda dependency feature of the CWL reference runner to obtain
```

Comment:
This is quite interesting. How are these environments created, in particular if there are other dependencies (so actually providing a complete environment.yml instead of a single dependency? Is that feature documented somewhere (if yes, a link would be great here). Is that part of the standard, or only part of the reference implementation?

Reply:
The CWL reference runner and some other runners based upon it (like `toil-cwl-runner` use a library from Galaxy to create the conda environments.

See https://docs.galaxyproject.org/en/latest/admin/conda_faq.html for documentation about that library's conda mode from a Galaxy perspective. FYI: When `cwltool` calls the Galaxy tool resolver library in conda mode, we set [both `conda_auto_install` and `conda_auto_init` to `true`](https://github.com/common-workflow-language/cwltool/blob/8ef515037de411abd2f84b569ad4d4a4f7a2c7a0/cwltool/software_requirements.py#L79)

If multiple conda packages are required for a particular `CommandLineTool` then use multiple entries under `SoftwareRequirement.packages`, one for each package.

Reply:
Is there an option to provide an environment.yml (because there this information is already stored and can be used when the modules is developed without the cwl interface)